test/run

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
```

Comment:
Please drop the `-e` here and add `set -eu` instead. That way, if you try to call this script like `bash test/run` for some reason, the options still apply. (Not that relevant here, but it's a good pattern to get used to).

The `-u` will also assert our assumptions about `$TEST_OS` always being set.